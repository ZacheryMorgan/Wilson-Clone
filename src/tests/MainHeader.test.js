import React from "react";
import { render } from "@testing-library/react";
import MainHeader from "../components/Main/MainHeader";

describe("MainHeader component", () => {
  it("MainHeader component renders correctly", () => {
    const { container } = render(<MainHeader />);
    expect(container).toMatchSnapshot();
  });
});
