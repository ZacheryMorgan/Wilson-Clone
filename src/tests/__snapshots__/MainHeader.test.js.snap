// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader component MainHeader component renders correctly 1`] = `
<div>
  <div
    class="main-header
 flex flex-col items-center relative justify-center
 "
  >
    <div
      class="main-header-img"
    >
      <picture>
        <source
          media="(min-width: 770px)"
          srcset="./assets/ShoreBg.jpg"
        />
        <img
          alt=""
          src="./assets/ShoreBGSmall.jpg"
        />
      </picture>
    </div>
    <div
      class="main-header-text
 text-center border-gray-300 border py-3 w-full
 "
    >
      <p
        class="title text-xl text-5 font-extrabold uppercase pb-2 tracking-wider"
      >
        A tradition unlike any other
      </p>
      <p
        class="sub tracking-wider text-xs pb-4"
      >
        Celebrate the season's first Shell major with Beach-approved shells and shells.
      </p>
      <button
        class="border-[#131b23] border-2 bg-white text-[#131b23] hover:bg-[#131b23] hover:text-white
 py-1 px-5 tracking-wider font-semibold text-lg uppercase"
      >
        Shop Shells
      </button>
    </div>
  </div>
</div>
`;
